plugins/backend/catalog-backend-module-aws/src/providers/AWSEC2Provider.test.ts

```diff
@@ -237,4 +237,72 @@ describe('AWSEC2Provider', () => {
       });
     });
   });
+
+  describe('where there is an instance with dynamic account config', () => {
```

Comment:
![Passing: test](https://gh-labels.dancrews.com/passing/test.svg)
This was untested.